test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -212,3 +215,154 @@ TEST_F(BlockQueryTest, GetTransactionsWithInvalidTxAndValidTx) {
   });
   ASSERT_TRUE(wrapper.validate());
 }
+
+/**
+ * @given block store with 2 blocks totally containing 3 txs created by
+ * user1@test AND 1 tx created by user2@test
```

Comment:
Imo as long as such details are inside the make_test_subscriber don't need to state it

Reply:
These details are in fixture, not in wrapper. Every test specifies given, these are not exceptions